alfasim_sdk/cli.py

```diff
@@ -45,16 +47,40 @@ def main():
 @click.option(
     "--author-name",
     prompt="-- Author Name",
-    help="Name of the plugin author to be displayed",
+    help="Name of the plugin author, this value is stored in plugin metadata and not displayed on the application explicitly",
 )
 @click.option(
     "--author-email",
     prompt="-- Author Email",
-    help="Email of the plugin author to be displayed",
+    help="Email of the plugin author, this value is stored in plugin metadata and not displayed on the application explicitly",
 )
 def template(dst, caption, plugin_id, author_name, author_email):
     """
-    Console script for alfasim_sdk.
+    Generate a template with the necessary files and structure to create a plugin.
+
+    The template folder will be placed on the ``dst`` option, that per default the current directory from where the command
+    was invoked.
+
+    The files generated and their contents are ready to be used, or customized and have the following structured.
```

Comment:
Misplaced comma and typo in structured: _are ready to be used or customized and have the following structure:_